types/port.pp

```diff
@@ -1,7 +1,7 @@
 # @summary
 #   Represents a port expression to be used within a rule.
 type Nftables::Port = Variant[
-  Array[Stdlib::Port, 1],
+  Array[Variant[Nftables::Port::Range, Stdlib::Port], 1],
```

Comment:
Would be good to add a couple of tests:

* And example in  of `nftables::rule` in https://github.com/voxpupuli/puppet-nftables/blob/master/spec/defines/simplerule_spec.rb
* The type itself in https://github.com/voxpupuli/puppet-nftables/blob/master/spec/type_aliases/nftables_port_spec.rb

Reply:
I can write a quick check for the nftables::rule spec (the Rocky 9 checked failed for some reason?  There's no error message).  The mixed array example, though, seems a bit more complicated; I can look at it soon, but not today?

Reply:
I wrote the second spec check, everything passed this time, can we get this merged up soon?